geoalchemy2/functions.py

```diff
@@ -88,6 +91,7 @@ class ST_AsGeoJSON(with_metaclass(_GenericMeta, functions.GenericFunction)):
     feature version introduced in PostGIS 3."""
 
     name = "ST_AsGeoJSON"
+    inherit_cache = True
```

Comment:
this one might be OK since there's no additional state here that isnt derived from the arguments that are already part of the cache key, but if you are in a hurry and dont have time to test very carefully, I'd keep inherit_cache=False

Reply:
I think the SQL will only depend on the arguments so I guess they should be part of the cache key. Let's try to keep True for this one.